app/__init__.py

```diff
@@ -4,14 +4,12 @@
 
 
 # local imports
-# from instance.config import app_config
 from app.api.v1.views import Home, Orders, MyOrder
 from app.api.v1.auth import Reg, Login, Refresh
-from app.api.v2.auth import Signup, Loginv2
-# from app.api.v2.menu_view import Menu, TestMe, MenuItem
```

Comment:
Remove commented code. Then I think we will be good.

Reply:
Done